test/test_cost.py

```diff
@@ -2867,6 +2867,8 @@ def __init__(self, actor_network, qvalue_network):
         loss.qvalue_network_params.items(include_nested=True, leaves_only=True)
     ):
         p1 = value
+        if isinstance(loss.actor_network, SafeProbabilisticModule):
+            key = ("module", *key) if isinstance(key, tuple) else ("module", key)
```

Comment:
This feels a bit messy, the actor network parameters have a different structure to the q-value network because they've be wrapped with a `SafeProbabilisticModule`. 

I think that extra layer is avoidable, but maybe we could wrap the value network with a `SafeProbabilisticModule` too using `Delta` distributions in order that the parameter structure remains in sync?